src/opentimelineio/imageSequenceReference.cpp

```diff
@@ -86,7 +88,13 @@ ImageSequenceReference::ImageSequenceReference(std::string const& target_url_bas
             sign = "-";
         }
 
-        std::string out_string = _target_url_base + _name_prefix + sign + zero_pad + image_num_string + _name_suffix;
+        // If the base does not include a trailing slash, add it
+        std::string path_sep = std::string();
+        if (_target_url_base.compare(_target_url_base.length() - 1, 1, "/") != 0) {
+            path_sep = "/";
+        }
+
+        std::string out_string = _target_url_base + path_sep + _name_prefix + sign + zero_pad + image_num_string + _name_suffix;
```

Comment:
Will hard coding "/" work on Windows? Perhaps `path_sep` could be defined at compile time based on OS.
Also, what would happen if `_target_url_base` is empty?

Reply:
So `os.sep` isn't quite the correct behavior because these are URLS, not paths. Also, my understanding is that Windows is actually tolerant of using `/` in place of `\`. So while it feels like it could be better, I think this method should probably work well without us writing too much path logic.

Reply:
Ah, well in that case there shouldn't be an issue. 
We can always come back to this if it turns out to be a problem.